.github/workflows/test.yml

```
@@ -0,0 +1,28 @@
+permissions:
+  contents: read
+name: "Test Empty Repos Scanner"
+
+on:
+  pull_request:
+    branches: [ main ]
+  push:
+    branches: [ main ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout repository 📂
+        uses: actions/checkout@v4
+      
+      - name: Setup Node.js 🔧
+        uses: actions/setup-node@v4
```

```suggestion
        uses: actions/setup-node@v4.4.0
```
This action doesn't move their tags when new minor or patch versions are released so lets follow the latest patch